src/main/resources/db/migration/common/V1_209__alter_transcription_table.sql

```diff
@@ -0,0 +1,17 @@
+--v46 Removed workflow related elements from transcription.
+--v47 Removed column transcription.current_state
+--    added column transcription.trs_id and FK to transcription_status
+
+ALTER TABLE transcription DROP COLUMN requested_by;
+ALTER TABLE transcription DROP COLUMN approved_by;
+ALTER TABLE transcription DROP COLUMN approved_on_ts;
+ALTER TABLE transcription DROP COLUMN transcribed_by;
```

Comment:
It's tricky - but if we're dropping these columns, we also need to drop the 3 constraints when comparing v43 with v47
* transcription_requested_by_fk
* transcription_approved_by_fk
* transcription_transcribed_by_fk

Reply:
If you drop a column with constraints the indexes and table constraints will automatically be dropped: https://www.postgresql.org/docs/7.4/sql-altertable.html#:~:text=DROP%20COLUMN,foreign%20key%20references%20or%20views.